### PR TITLE
first commit to pingcap/jepsen

### PR DESCRIPTION
**Pull request by UncP**
Signed-off-by: UncP <770778010@qq.com>

### CODE REVIEW

File: tidb/src/tidb/core.clj [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+  (reify db/DB
+    (setup! [_ test node]
+      (info node c/*host*)
+      (c/su
+        (cu/install-tarball! c/*host* tidb-url tidb-dir)
+      )
+
+      ; ./bin/pd-server --name=pd1 \
+      ;                 --data-dir=pd1 \
+      ;                 --client-urls="http://192.168.199.113:2379" \
```
Comment (siddontang) [inline tidb/src/tidb/core.clj]: why 192.168.199.113 here?

File: tidb/src/tidb/core.clj [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+        :--pd        (pd-cluster test)
+        :--addr      (str (name node) ":" "20160")
+        :--data-dir  (get-in tidb-map [node :kv])
+        :--log-file  (str "tikv.log")
+      )
+
+      ; ./bin/tidb-server --store=tikv \
+                        ; --path="192.168.199.113:2379,192.168.199.114:2379,192.168.199.115:2379" \
+                        ; --log-file=tidb.log
+      (cu/start-demon!
```
Comment (siddontang) [inline tidb/src/tidb/core.clj]: do we need to wait some times to guarantee PD and TiKV start successfully? 
If not, the TiDB may fail to start.

File: tidb/src/tidb/core.clj (hunk 1 above)
Comment (UncP) [inline tidb/src/tidb/core.clj]: it's a comment

File: tidb/src/tidb/core.clj (hunk 2 above)
Comment (UncP) [inline tidb/src/tidb/core.clj]: yes, we must, i will correct that when introduce mysql client to test.

File: tidb/src/tidb/core.clj (hunk 1 above)
Comment (UncP) [inline tidb/src/tidb/core.clj]: to show the format when initialize pd, kv, etc.

Comment (siddontang) [inline tidb/src/tidb/core.clj]: it is better to use n1:2379 here, n1-n5 is the default host name in jepsen.

Comment (UncP) [inline tidb/src/tidb/core.clj]: sure

File: tidb/src/tidb/core.clj (hunk 2 above)
Comment (siddontang) [inline tidb/src/tidb/core.clj]: so add wait time here, and you can add test in another PR

Comment (UncP) [inline tidb/src/tidb/core.clj]: sure

**Pull request closed without merging**